test/functional/rpc_rawtransaction.py

```diff
@@ -358,7 +358,7 @@ def run_test(self):
 
         # decoderawtransaction tests
         # witness transaction
-        encrawtx = "010000000001010000000000000072c1a6a246ae63f74f931e8365e15a089c68d61900000000000000000000ffffffff0100e1f50500000000000000000000"
+        encrawtx = "010000000001010000000000000072c1a6a246ae63f74f931e8365e15a089c68d61900000000000000000000ffffffff0100e1f50500000000000102616100000000"
```

Comment:
Could we prevent, somehow, that the old transaction without witness passes this test?

Reply:
Merging #14039 would prevent the test from passing.